src/stretchy.js

```diff
@@ -137,280 +137,232 @@ const katexImagesData = {
--107 64l-10 8H0v40zM0 157v40h399730v-40zm0 194v40h399730v-40z'/></svg>`,
+-107 64l-10 8H0v40zM0 157v40h399730v-40zm0 194v40h399730v-40z'/>`,
+
+    leftarrow: `<path d='M400000 241H110l3-3c68.7-52.7 113.7-120
+ 135-202 4-14.7 6-23 6-25 0-7.3-7-11-21-11-8 0-13.2.8-15.5 2.5-2.3 1.7-4.2 5.8
+-5.5 12.5-1.3 4.7-2.7 10.3-4 17-12 48.7-34.8 92-68.5 130S65.3 228.3 18 247
+c-10 4-16 7.7-18 11 0 8.7 6 14.3 18 17 47.3 18.7 87.8 47 121.5 85S196 441.3 208
+ 490c.7 2 1.3 5 2 9s1.2 6.7 1.5 8c.3 1.3 1 3.3 2 6s2.2 4.5 3.5 5.5c1.3 1 3.3
+ 1.8 6 2.5s6 1 10 1c14 0 21-3.7 21-11 0-2-2-10.3-6-25-20-79.3-65-146.7-135-202
+ l-3-3h399890zM100 241v40h399900v-40z'/>`,
+
+    rightarrow: `<path d='M0 241v40h399891c-47.3 35.3-84 78-110 128
+-16.7 32-27.7 63.7-33 95 0 1.3-.2 2.7-.5 4-.3 1.3-.5 2.3-.5 3 0 7.3 6.7 11 20
+ 11 8 0 13.2-.8 15.5-2.5 2.3-1.7 4.2-5.5 5.5-11.5 2-13.3 5.7-27 11-41 14.7-44.7
+ 39-84.5 73-119.5s73.7-60.2 119-75.5c6-2 9-5.7 9-11s-3-9-9-11c-45.3-15.3-85
+-40.5-119-75.5s-58.3-74.8-73-119.5c-4.7-14-8.3-27.3-11-40-1.3-6.7-3.2-10.8-5.5
+-12.5-2.3-1.7-7.5-2.5-15.5-2.5-14 0-21 3.7-21 11 0 2 2 10.3 6 25 20.7 83.3 67
+ 151.7 139 205zm0 0v40h399900v-40z'/>`,
+};
+
+const innerSVG = {
+    // Since bcancel's SVG is inline and it omits the viewBox attribute,
+    // it's stroke-width will not vary with span area.
+    bcancel: `<line x1='0' y1='0' x2='100%' y2='100%' stroke-width='0.046em'/>`,
+
+    cancel: `<line x1='0' y1='100%' x2='100%' y2='0' stroke-width='0.046em'/>`,
+
+    // The doubleleftarrow geometry is from glyph U+21D0 in the font KaTeX Main
+    doubleleftarrow: `><svg viewBox='0 0 400000 549'
+preserveAspectRatio='xMinYMin slice'>${svgPath["doubleleftarrow"]}</svg>`,
+
+    // doubleleftrightarrow is from glyph U+21D4 in font KaTeX Main
+    doubleleftrightarrow: `><svg width='50.1%' viewBox='0 0 400000 549'
+preserveAspectRatio='xMinYMin slice'>${svgPath["doubleleftarrow"]}</svg>
+<svg x='50%' width='50%' viewBox='0 0 400000 549' preserveAspectRatio='xMaxYMin
+ slice'>${svgPath["doublerightarrow"]}</svg>`,
+
+    // doublerightarrow is from glyph U+21D2 in font KaTeX Main
+    doublerightarrow: `><svg viewBox='0 0 400000 549'
+preserveAspectRatio='xMaxYMin slice'>${svgPath["doublerightarrow"]}</svg>`,
 
     // hookleftarrow is from glyph U+21A9 in font KaTeX Main
     hookleftarrow: `><svg width='50.1%' viewBox='0 0 400000 522'
-preserveAspectRatio='xMinYMin slice'><path fill='currentcolor' d='M400000 241
-H110l3-3c68.7-52.7 113.7-120 135-202 4-14.7 6-23 6-25 0-7.3-7-11-21-11-8 0
--13.2.8-15.5 2.5-2.3 1.7-4.2 5.8-5.5 12.5-1.3 4.7-2.7 10.3-4 17-12 48.7-34.8 92
--68.5 130S65.3 228.3 18 247c-10 4-16 7.7-18 11 0 8.7 6 14.3 18 17 47.3 18.7
- 87.8 47 121.5 85S196 441.3 208 490c.7 2 1.3 5 2 9s1.2 6.7 1.5 8c.3 1.3 1 3.3 2
- 6s2.2 4.5 3.5 5.5c1.3 1 3.3 1.8 6 2.5s6 1 10 1c14 0 21-3.7 21-11 0-2-2-10.3-6
--25-20-79.3-65-146.7-135-202l-3-3h399890zM100 241v40h399900v-40z'/></svg>
+preserveAspectRatio='xMinYMin slice'>${svgPath["leftarrow"]}</svg>
```

Comment:
Even though this change doesn't save much after minification and gzip compression, it's good to de-duplicate strings so that if we have to modify the path there's only one place to modify it.